src/components/Header/index.jsx

```diff
@@ -68,20 +68,33 @@ const GordonHeader = ({ onDrawerToggle }) => {
   }, [width]);
 
   const createDialogBox = () => {
-    const isOffline = dialog === 'offline';
-    return (
-      <GordonDialogBox
-        open={dialog}
-        onClose={() => setDialog(null)}
-        title={isOffline ? 'Unavailabile Offline' : 'Login Required'}
-        buttonClicked={() => setDialog(null)}
-        buttonName={'Okay'}
-      >
-        {isOffline
-          ? 'That page is not available offline. Please reconnect to internet to access this feature.'
-          : 'That page is only available to authenticated users. Please log in to access it.'}
-      </GordonDialogBox>
-    );
+    if (dialog === 'offline') {
+      return (
+        <GordonDialogBox
+          open={dialog}
+          onClose={() => setDialog(null)}
+          title={'Offline Mode'}
+          buttonClicked={() => setDialog(null)}
+          buttonName={'Okay'}
+        >
+          This feature is unavailable offline. Please reconnect to internet to access this feature.
+        </GordonDialogBox>
+      );
+    } else if (dialog === 'unauthorized') {
+      return (
+        <GordonDialogBox
+          open={dialog}
+          onClose={() => setDialog(null)}
+          title={'Credentials Needed'}
+          buttonClicked={() => setDialog(null)}
+          buttonName={'Okay'}
+        >
+          This feature is unavailable while not logged in. Please log in to access it.
+        </GordonDialogBox>
+      );
+    } else {
+      return null;
+    }
```

Comment:
Is there a reason this was changed from the previous version? The logic looks identical?

Reply:
I don't remember why it was changed (I don't remember changing it, maybe it was from a merge? I'm not sure) but regardless I can switch it back